modules/qualitycontrol/SendVideoController.js

```diff
@@ -1,28 +1,35 @@
 import * as JitsiConferenceEvents from '../../JitsiConferenceEvents';
+import RTCEvents from '../../service/RTC/RTCEvents';
 import MediaSessionEvents from '../xmpp/MediaSessionEvents';
 
 /**
- * The class manages send and receive video constraints across media sessions({@link JingleSessionPC}) which belong to
+ * The class manages send video constraints across media sessions({@link JingleSessionPC}) which belong to
  * {@link JitsiConference}. It finds the lowest common value, between the local user's send preference and
  * the remote party's receive preference. Also this module will consider only the active session's receive value,
  * because local tracks are shared and while JVB may have no preference, the remote p2p may have and they may be totally
  * different.
  */
-export class QualityController {
+export class SendVideoController {
     /**
      * Creates new instance for a given conference.
      *
      * @param {JitsiConference} conference - the conference instance for which the new instance will be managing
-     * the quality constraints.
+     * the send video quality constraints.
+     * @param {RTC} rtc - the rtc instance that is responsible for sending the messages on the bridge channel.
      */
-    constructor(conference) {
+    constructor(conference, rtc) {
         this.conference = conference;
+        this.layerSuspensionEnabled = conference.options?.config?.enableLayerSuspension;
+        this.rtc = rtc;
         this.conference.on(
             JitsiConferenceEvents._MEDIA_SESSION_STARTED,
             session => this._onMediaSessionStarted(session));
         this.conference.on(
             JitsiConferenceEvents._MEDIA_SESSION_ACTIVE_CHANGED,
             () => this._propagateSendMaxFrameHeight());
+        this.rtc.on(
+            RTCEvents.SENDER_VIDEO_CONSTRAINTS_CHANGED,
+            videoConstraints => this._propagateSendMaxFrameHeight(videoConstraints));
```

Comment:
I think it would be a bit cleaner if this part was split into:
```suggestion
            videoConstraints => {
                this._senderVideoConstraints = videoConstraints;
                this._propagateSendMaxFrameHeight());
             }
```

Reply:
also please change `this.senderVideoConstraints` to `this._senderVideoConstraints` in other places in this class